sys-libs/libcxxabi/libcxxabi-3.9.0.ebuild

```diff
@@ -72,6 +72,11 @@ multilib_src_configure() {
 			-DLIT_COMMAND="${EPREFIX}"/usr/bin/lit
 		)
 	fi
+	if use elibc_musl; then
+		mycmakeargs+=(
+			-DCMAKE_CXX_FLAGS="-D_LIBCPP_HAS_MUSL_LIBC=1"
```

Comment:
Don't play with that. Just set, on top:

    local -x CPPFLAGS="${CPPFLAGS} -D..."

Is it fixed in -9999? If not, we probably have to prepare a fix for CMakeLists.